aws/efs/README.md

```diff
@@ -5,9 +5,109 @@
 quay.io/external_storage/efs-provisioner:v0.1.0
 ```
 
+The efs-provisioner is a storage plugin for kubernetes. It consists of a container that has access to an AWS [EFS](https://aws.amazon.com/efs/) resource. The container reads a configmap which contains the EFS filesystem ID, the AWS region and the name you want to use for your efs-provisioner. This name will be used later when you create a storage class.
```

Comment:
storage plugin isn't right, at best this can be called a subset of a storage plugin. storage plugin implies a full flex volume plugin or container storage interface implementation which are much more than this.